### DISCUSSION TITLE
Support multiple applications in monorepo

### DISCUSSION THREAD

**Discussion by localusercamp** [nuxtrdev/nuxtr-vscode#68] [Ideas]
Now we have this 
![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/45385480/62f997de-786b-44f9-89ae-1cf9dad542c5)

But what if i have 2 nuxt-apps in my monorepo?

**Comment by adhamfarrag** [comment]
Monorepo mode is under development and configuring multiple Nuxt apps will be supported.

**Reply by vinceglb** [reply]
Hi! Is there any update about this?
